sns/config/sns_proposal_summary.md

```diff
@@ -32,9 +32,9 @@ For more information on the initial token allocation please [read here](https://
 
 # Governance
 
-- Each decentralization sale participant (including the commnity fund) will receive a basket of 5 equal value CHAT neurons with dissolve delays of 0, 3, 6, 9 and 12 months respectively
-- The founding dev team will receive a basket of 5 neurons, the first with zero dissolve delay, and the rest with a dissolve delay of 1 month but with vesting periods of 1, 2, 3 and 4 years respectively
-- DFINITY will receive a basket of 4 neurons all with a dissolve delay of 1 month but with vesting periods of 6, 12, 18 and 24 months respectively
+- Each decentralization sale participant (including the commnity fund) will receive their tokens in a basket of 5 equal value CHAT neurons with dissolve delays of 0, 3, 6, 9 and 12 months respectively
```

Comment:
I think we should capitalise 'Community Fund' too (as well as fixing the typo)

Reply:
More haste less speed!